pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -55,6 +62,13 @@ var (
 		metrics.ALPHA,
 		"")
 
+	containerSwapUsageDesc = metrics.NewDesc("container_swap_usage_bytes",
```

Comment:
Can we just update some of the comments / docs (like https://github.com/kubernetes/kubernetes/pull/118865/files#diff-4cc35a79a447d7cd52ad099092ca90d44b92ff38e0bd266dd2a0b9c141b504a3R62) to indicate that this will only be reported for Linux?
I agree that it would be cleaner to postfix with an OS for the metrics endpoints

Reply:
do we need to disable reporting completely on Windows?

Reply:
that would probably be a good idea and less confusing to users

Reply:
personally I don't love the postfix. I think the doc should be clear enough. WDYT?

Reply:
yes, looking at it I don't like it either. Maybe just do not report those metrics on Windows.

Reply:
@SergeyKanzhelev @haircommander

I agree, I don't the postfix either.
It's now removed. We don't report on windows + I've added docs and help strings to reflect that.